system-addon/test/unit/asrouter/templates/SimpleSnippet.test.jsx

```diff
@@ -18,6 +18,13 @@ function mountAndCheckProps(content = {}) {
 }
 
 describe("SimpleSnippet", () => {
+  let sandbox;
```

Comment:
did you leave this in accidentally? tests seem to still pass when I remove this chunk

Reply:
Oh. Yes.. moved the tests in asrouter-content.test.jsx and forgot to cleanup.